app/routes/admin/permissions/system-roles.js

```diff
@@ -5,5 +5,15 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.l10n.t('System Roles');
+  },
+  model() {
+    return this.get('store').findAll('user-permission');
+  },
+  actions: {
+    willTransition() {
+      this.get('controller.model').forEach(function(userPermission) {
```

Comment:
@niranjan94 um.. do you mean should  I use something of the  sort 
`var that = this`?